metroplex.js

```diff
@@ -192,18 +162,13 @@ Metroplex.readable('disconnect', function disconnect(spark) {
  * @returns {Metroplex}
  * @api public
  */
-Metroplex.readable('servers', function servers(self, fn) {
+Metroplex.readable('servers', function servers(fn) {
   var metroplex = this;
 
-  if ('boolean' !== typeof self) {
-    fn = self;
-    self = 0;
-  }
-
-  this.redis.smembers(this.namespace +'servers', function smembers(err, members) {
-    if (self) return fn(err, members);
-
-    fn(err, (members || []).filter(function filter(address) {
+  metroplex.redis.keys(metroplex.namespace + 'server:*', function keyList(err, list) {
```

Comment:
Returning ALL the servers can be useful, for example for debugging, so  I would add back the `self` argument.